src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,18 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
     public static void main(String[] args) {
```

Comment:
Error:  /home/runner/work/jv-logger/jv-logger/src/main/java/mate/academy/Main.java:14: 'METHOD_DEF' should be separated from previous statement. [EmptyLineSeparator]